src/Emulator/Visuals/Visuals.js

```diff
@@ -1,11 +1,236 @@
-import React from 'react';
+import React, {useRef, useState, useEffect} from 'react';
 import './Visuals.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearchPlus, faSearchMinus, faExpand, faEyeSlash} from '@fortawesome/free-solid-svg-icons';
+import Draggable from 'react-draggable';
+import { parse } from 'path';
 
 const Visuals = () => {
+  const containerHeight = '50vh';
+  const containerWidth = '75vw';
```

Comment:
I'm actually not sure the best way to define this, but I suspect this would be part of the parent container which holds the button, and this container will be 100% within that.